src/test/testdata/expected/lsl/annotations.lsl

```diff
@@ -9,7 +9,7 @@ annotation Anno(
 );
 automaton A : Int {
```

Comment:
Maybe in future divide this test-case on three files ? "combinationOfAnnotations", "functionAnnotations", "fieldsAnnotation" and etc... I assume that this file must have name "functionAnnotations" because only functions and it's arguments have annotations

Reply:
Since we already have "functionAnnotations", I decided to rename this file to "initValuesAnnotations" and keep here only annotations with initial values. I will add other variants in the future